src/components/ForecastSummary.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Forecast from "./Forecast";
+import { getDay } from "./DataFunction";
+
+export default function Card({ userInput }) {
```

Comment:
Its not good practice to call the component something different to the name of the file, this should be called `ForecastSummary`